server/src/main/proto/spine_examples/kanban/commands.proto

```diff
@@ -36,15 +36,37 @@ option java_multiple_files = true;
 
 import "spine_examples/kanban/identifiers.proto";
 import "spine_examples/kanban/wip_limit.proto";
+import "spine_examples/kanban/column_position.proto";
 
 // A command to create a new board.
 message CreateBoard {
 
     // The ID of the board to create.
+    //
+    // Example note: By convention the first field of a command is its target.
+    // So we don't have to mark it as required.
     BoardId board = 1;
 }
 
-// A command to create a new column on a board.
+// A command to add a new column on a board.
+message AddColumn {
+
+    // The ID of the board on which to create the column.
+    BoardId board = 1;
+
+    // The ID of the column to add.
+    ColumnId column = 2 [(required) = true];
+
+    // The name of the column.
+    string name = 3 [(required) = true];
+
+    // The desired position of the column.
+    //
+    // An actual position where the column is placed may differ from the desired one.
+    ColumnPosition desired_position = 4 [(required) = true];
+}
+
+// A command to create a column.
 message CreateColumn {
```

Comment:
I would note here that this command is internal to the Bounded Context, and is not intended to be sent by actors from outside. 

We have a Proto option for that. Please refer to "options.proto". And let's discuss. 

I think, same applies to `PlaceColumn`. And perhaps to other "internal" commands as well.